### DISCUSSION TITLE
Streamlink on VLC archlinux doens't show image/audio for twitch.

### DISCUSSION THREAD

**Discussion by zachvlat** [streamlink/streamlink#3808] [General]
As the title says, when i open a twitch link through terminal I don't see anything not hearing any sound. 
I m attaching system info and the actual screen of my vlc if you need any more info let me know. The version I m using is the 2.1.1.

![vlcsnap-2021-06-20-09h44m49s060](https://user-images.githubusercontent.com/28532979/122665607-dc92f580-d1b0-11eb-8514-91ad7e94ae75.png)

![Screenshot from 2021-06-20 10-19-18](https://user-images.githubusercontent.com/28532979/122665629-00563b80-d1b1-11eb-8dce-bfdb428b7d77.png)

**Comment by bastimeyer** [comment]
Not a Streamlink issue and instead a VLC configuration issue on your system. I have therefore moved your thread to the discussions forum.

You can try checking the player's stdout/stderr via [`--verbose-player`](https://streamlink.github.io/cli.html#cmdoption-verbose-player) for any error messages. Maybe try disabling GPU acceleration in VLC, or use a different player, like MPV for example, but apart from that, decoding/rendering issues in your player have to be solved by yourself.

This is not a problem with Streamlink.
